### DISCUSSION TITLE
Packaging (tracker bug)

### DISCUSSION THREAD

**Discussion by teolemon** [openfoodfacts/robotoff#1242] [Ideas]
### Autoselect with Regex
- [ ]  #260 
  - [ ] #301 (Deployed, but does not seem to work ??)
### Extract with Regex
- [ ] Extract full recycling instructions with Regex
  - [ ] Extract packaging shape with Regex
  - [ ] Extract packaging material with Regex
  - [ ] Extract packaging shapes and materials with Regex
- [ ] Copy producer recycling instructions to our own recycling instructions field
  - [x] https://github.com/openfoodfacts/robotoff/issues/257
### More advanced solutions (openfoodfacts-ai)
- [x] https://github.com/openfoodfacts/openfoodfacts-ai/issues/207
  - [ ] Those are 3 models, probably solving the 3 issues below
  - [ ] https://github.com/openfoodfacts/openfoodfacts-ai/issues/56
  - [ ] https://github.com/openfoodfacts/openfoodfacts-ai/issues/29
  - [ ] https://github.com/openfoodfacts/openfoodfacts-ai/issues/23
- [ ] https://github.com/openfoodfacts/openfoodfacts-ai/issues/72
  - [ ] Those have probably been detected by Logos & Labels, can we make a Hunger Games dashboard out of them ?
- [ ] https://github.com/openfoodfacts/openfoodfacts-ai/issues/55

### Part of
- #374
- https://github.com/openfoodfacts/openfoodfacts-server/issues/6374
- https://github.com/openfoodfacts/openfoodfacts-ai/issues/97

**Comment by teolemon** [comment]
### Issues with Regex extraction
https://world.openfoodfacts.org/product/3700007900439/les-tomates-cerises
![image](https://user-images.githubusercontent.com/1689815/75525861-e79ca200-5a10-11ea-83af-172a87965e84.png)
>>Packaging: Barquette en plastique, Film en plastique
```
NET-PLASTIC recycle with carrier\nbags at larger stores- not at kerbside\nTAG-MIXED MATERIAL\nnot currently recycled\n19 Sept\nGrown by\nAntonio Peregrin, Spain\nCount: X4\nR0466B\nL-81-06\n7500735\n0914\nAt home, store in a cool, dry place.
``` 
![image](https://user-images.githubusercontent.com/1689815/76164601-84b5b400-6150-11ea-93ed-4be7dd41c8f9.png)
https://world.openfoodfacts.org/images/products/01795257/1.jpg

**Comment by raphael0202** [comment]
The first bug is fixed: ee71a6652663bc14170dc4a5c1830f841bd96e86.
Can you tell me more about the second image?
